File: frontend-web/publisher-app/src/app/shared/models/Comment.ts
export class PostComment {
    id: number;
    postId: number;
    content: string | null | undefined;
    commenter: string;
    creationDate: string;

    constructor(postId: number, content: string, commenter: string, id: number, creationDate: string) {
        this.postId = postId;
        this.content = content;
        this.commenter = commenter;
        this.id = id;
        this.creationDate = creationDate;
    }
}

File: backend-java/SpringCloudPXL/comment-service/src/main/java/be/pxl/services/domain/Comment.java
package be.pxl.services.domain;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDate;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private long postId;
    @Column(columnDefinition = "LONGTEXT")
    private String content;
    private String commenter;
    private LocalDate creationDate;
}
